core/geometry/src/polyface/IndexedPolyfaceVisitor.ts

```diff
@@ -230,7 +230,7 @@ export class IndexedPolyfaceSubsetVisitor extends IndexedPolyfaceVisitor {
    * * The activeFacetIndices array indicates all facets to be visited.
    */
   public static createSubsetVisitor(polyface: IndexedPolyface, activeFacetIndices: number[], numWrap: number): IndexedPolyfaceSubsetVisitor {
-    return new IndexedPolyfaceSubsetVisitor(polyface, activeFacetIndices.slice(), numWrap);
```

Comment:
Any idea why this was copying the input array before?

Reply:
No idea. I noticed the slice occurs in the constructor, so it's not needed here.